### PR TITLE
/akn FRBR URI prefix

### LINKED ISSUE #17: /akn FRBR URI prefix
AKN 3 requires that AKN FRBR URIs start with `/akn/`. Non-AKN FRBR URIs must start with some other prefix.

I think Cobalt will need to support both the old style (`/xx[-locality]/`) and new-style (`/akn/`), at least for a short while.

## Proposal

1. Cobalt must be able to parse FRBR URIs that start with `/akn/`
2. Cobalt should assume that FRBR URIs that don't start with `/akn/` but do match the `/xx[-locality]/` initial format are valid, un-prefixed FRBR URIs. ie. be backwards-compatible with what we do today.
3. Cobalt should throw an error for prefixed FRBR URIs that it doesn't understand (eg. `/ukpga/...`).
3. When turning a parsed FRBR URI into a string, it should by default include the `/akn/` prefix. We should be able to disable this, perhaps by setting `frbr_uri.prefix = None`.

These will have to be guided by our use cases.

### PR DESCRIPTION
**Pull request by goose-life**
will close #17

### CODE REVIEW

File: cobalt/uri.py [hunk 1]
```diff
@@ -131,11 +137,16 @@ def uri(self):
 
     def work_uri(self, work_component=True):
         """ String form of the work URI. """
+        prefix = self.prefix
```
Comment (longhotsummer) [inline cobalt/uri.py]: no real benefit to assigning this to another variable, just use `self.prefix` later on

**Comment by longhotsummer** [comment]
Great. Backwards- and forwards-compatible.

**Comment by longhotsummer** [comment]
It'll be useful to including documentation changes to the README for each of these release. File it under Cobalt 4.0 (release date TBD).

**Comment by goose-life** [comment]
not too confident on the docs, feedback welcomed!

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**